### ISSUE TITLE
Cannot view stats when server is down.

### ISSUE THREAD

**Issue by colinmollenhour** [munin-monitoring/munin#706]
Here is an example of a server that has been down for about 30 minutes. I'd like to see the stats right before the server went down to see if there could be a load issue, but I cannot view the stats at all: http://screencast.com/t/YnCb0sd3

**Comment by ssm** [comment]
This depends on #720

**Comment by sumpfralle** [comment]
Duplicate of #579
